diff --git a/LockScreen.qml b/LockScreen.qml
index 071b323..61e368e 100644
--- a/LockScreen.qml
+++ b/LockScreen.qml
@@ -176,8 +176,8 @@ ApplicationWindow {
                 updatesEnabled: lockScreen.locked
                 color: lockScreen.textColor
                 opacity: systemStarted.value && vignette.opened && Desktop.deviceLockState <= DeviceLock.Locked
-                         ? (deviceLockContainer.isCurrentItem ? deviceLockItem.opacity : lockItem.clock.transitionOpacity)
-                         : 0.0
+                         ? (deviceLockContainer.isCurrentItem ? deviceLockItem.opacity : 1.0)
+                         : 1.0
                 z: 1
             }
 
